interface/themes/colors/openemr5/recall-flow-board.scss

```diff
@@ -0,0 +1,16 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~openemr5 recall and flowboard *~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+.divTable, .borderShadow {
+    background: $palepale !important;
+    border: 0.5px solid $pale !important;
+    box-shadow: 0px 0px 0px $pale !important;
+    border-radius: 0px !important;
+}
+.divTableCell, .divTableHead {
+    border: 1px solid $mid !important;
+    border-radius: 0px !important;
+}
+.divTableHeading {
+    background-color: $pale !important;
+}
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~openemr5 recall and flowboard *~~~~~~~~~~~~~~~~~~~~~~~~~*/
```

Comment:
Just checking on this. Does this mean, this css stuff only ends up in the color css themes?(since in the colors directory?)
Is this being used on the recall sheet in the current codebase?

Reply:
These are specific for flat stylesheets in order to seamlessly blend with other pages on the application.  Light and Manila use what the original developers had used😀